apps/src/code-studio/pd/foorm/FoormEditor.jsx

```diff
@@ -56,14 +33,57 @@ class FoormEditor extends React.Component {
 
     this.state = {
       formKey: 0,
-      formPreviewQuestions: null
+      formPreviewQuestions: null,
+      num_facilitators: 2,
```

Comment:
We could camel case these field names, but also seems find to leave them in their expected final form..

Reply:
The individual facilitators I would need to leave in underscore format since they get passed as-is, but I could change the others. Don't have a strong preference